packages/legend-studio/src/stores/ViewerStore.tsx

```diff
@@ -198,6 +199,16 @@ export class ViewerStore {
             undefined,
           )) as Entity[];
       }
+
+      // Get Project Config
+      const projectConfiguration: PlainObject<ProjectConfiguration> =
+        flowResult(
+          yield this.editorStore.applicationStore.networkClientManager.sdlcClient.getConfiguration(
```

Comment:
Did we account for viewer mode where workspaceId would not be defined like through a version or revisionID?

Reply:
Nope, the logic within `ProjectConfigurationEditor` component always expects `projectConfiguration` to be present in the store. `EditorStore` does load the `projectConfiguration` in standard mode while `ViewerStore` doesn't, hence the crash on the app whenever user try to view project `config`  in viewer mode.

Reply:
yeah understood, my question was more when no workspace is provided, which occurs when you are in viewer mode for released version or on specific revision.

Reply:
I couldn't find any logic that handles that, so I assumed we don't

Reply:
lets test your changes locally with a revision/version id and make sure this still works.